test/e2e/fixtures.go

```diff
@@ -474,3 +475,34 @@ func createTestBusyboxPods(tb testing.TB, data *TestData, num int, ns string, no
 
 	return podNames, podIPs, cleanupFn
 }
+
+// testServiceFromNode is a helper function which uses polling mechanism
+// to make sure that Service has been realized.
+func testServiceFromNode(t *testing.T, url, nodeName string, timeout time.Duration, statusReady bool) error {
+	cmd := fmt.Sprintf("curl -s --connect-timeout 1 --retry-connrefused %s", url)
+	err := wait.Poll(1*time.Second, timeout, func() (bool, error) {
+		t.Logf(cmd)
+		rc, stdout, stderr, err := RunCommandOnNode(nodeName, cmd)
+		switch statusReady {
+		case true:
+			if rc == 0 {
+				return true, nil
+			} else {
```

Comment:
you can remove `else` like below which help to reduce unnecessary indent.
```
			if rc == 0 {
				return true, nil
			}
			t.Logf("stdout: %s - stderr: %s - err: %v", stdout, stderr, err)
			// keep trying.
			return false, nil
```

Reply:
done.